### PR TITLE
Enable 'tx' command for green-liquid-cli

### PR DESCRIPTION
**Pull request by jkauffman1**

### CODE REVIEW

File: green_cli/liquid.py [hunk 1]
```diff
@@ -81,5 +107,59 @@ def sendtoaddress(session, details):
...
+        click.echo(f"{asset}:")
+        click.echo(f"\tavailable: {available_per_asset[asset]}")
+        click.echo(f"\tused: {used_per_asset[asset]}")
+        click.echo(f"\tchange: {change_per_asset[asset]}")
+
+    click.echo(f"size: {tx['transaction_size']}")
+    click.echo(f"vsize: {tx['transaction_vsize']}")
+    click.echo(f"weight: {tx['transaction_weight']}")
+    click.echo(f"fee: {tx['fee']}")
+    click.echo(f"fee rate: {tx['calculated_fee_rate']} sat/kb")
```
Comment (jgriffiths) [inline green_cli/liquid.py:137]: nit: lsat/kb

File: green_cli/liquid.py [hunk 2]
```diff
@@ -81,5 +107,59 @@ def sendtoaddress(session, details):
...
+    if output['is_fee']:
+        fg = 'red'
+        if not options['show_all'] and not options['show_fee']:
+            return
+    elif output['is_change']:
+        fg = 'green'
+        if not options['show_all'] and not options['show_change']:
+            return
+    else:
+        if not options['show_all'] and options['show_fee'] or options['show_change']:
```
Comment (jgriffiths) [inline green_cli/liquid.py:152]: brackets would be helpful here to explicitly show the order of and/or

File: green_cli/liquid.py [hunk 3]
```diff
@@ -81,5 +107,59 @@ def sendtoaddress(session, details):
...
+            return
+
+    value = output['satoshi']
+    asset = _asset_name(output['asset_id'])
```
Comment (jgriffiths) [inline green_cli/liquid.py:156]: nit `asset_name` is a better var name here IMO

File: green_cli/tx.py [hunk 4]
```diff
@@ -149,30 +150,32 @@ def setfeerate(session, feerate):
     with Tx(allow_errors=True) as tx:
         tx['fee_rate'] = feerate
 
+def _print_tx_output(options, output):
+    if show_all or (output['is_change'] == show_change):
```
Comment (jgriffiths) [inline green_cli/tx.py:154]: I think `show_change` should be passed as a param here

**Comment by jgriffiths** [comment]
utack c9983839ff738004d37dea65453a86f0d75a95cc with nits

File: green_cli/liquid.py (hunk 1 above)
Comment (jkauffman1) [reply green_cli/liquid.py]: done thanks

File: green_cli/liquid.py (hunk 3 above)
Comment (jkauffman1) [reply green_cli/liquid.py]: done, thanks

**Pull request merged**